src/test/com/extrahardmode/service/TestMultiWorldConfig.java

```diff
@@ -21,92 +21,81 @@
 
```

Comment:
why'd you remove this test?

Reply:
Because the test is mainly testing `loadNode` which is removed.
ofcourse we have to write test for other methods.